recipes/libsndfile/config.yml

```diff
@@ -1,3 +1,5 @@
 versions:
   "1.0.28":
-    folder: all
+    folder: "all"
```

Comment:
isn't 1.0.28 version located in "1.0.28" foler ?

Reply:
it was changed in a separate PR as I recall

Reply:
yes, this PR has to be rebased on top of master, now that https://github.com/conan-io/conan-center-index/pull/2596 is merged

Reply:
Rebase done. There is other remark from @prince-chrismc but I have no idea what to answer...

Reply:
The CCI team will see my comment and let you know if there's anything, it was a general question.

Feel free to make this thread as resolved =)